manifests/config.pp

```diff
@@ -40,10 +40,21 @@
     ensure  => 'present',
     shell   => $foreman_proxy::shell,
     comment => 'Foreman Proxy daemon user',
+    gid     => $foreman_proxy::group,
     groups  => $foreman_proxy::groups + $dns_groups + $puppet_groups,
     home    => $foreman_proxy::dir,
-    require => Class['foreman_proxy::install'],
-    notify  => Class['foreman_proxy::service'],
+    system  => true,
+  }
+
+  group { $foreman_proxy::group:
+    system => true,
+  }
+
+  # Provided by packaging by provides autorequire for files
+  file { $foreman_proxy::config_dir:
+    ensure => directory,
+    owner  => 'root',
+    group  => 0,
```

Comment:
any reason not to write `group => 'root'` here?

Reply:
Technically we still mark it compatible with FreeBSD and writing `0` is shorter than `BSD ? 'wheel' : 'root'`. I copied it from elsewhere in this file.

Reply:
"ooof", okay